.github/workflows/codespell.yml

```
@@ -2,7 +2,7 @@ name: CodeSpell
 on:
   - pull_request
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref || github.run_id }}
```

`github.ref` looks unnecessary. `github.head_ref` if PR, otherwise `github.run_id` suffices, right?

```suggestion
  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```